### PR TITLE
[IMP] mail plugins: add instructions to gmail plugin

### PR DESCRIPTION
**Pull request by meng-odoo**
Added instructions to the Gmail Plugin doc for Odoo on-premise users.
Resolves Issue [#3645](https://github.com/odoo/documentation/issues/3645)

### CODE REVIEW

**Comment by meng-odoo** [comment]
Hi @StraubCreative, this is ready for your review.

**Comment by jcs-odoo** [comment]
Hi @meng-odoo  :)
You can link issues and branches in their "development" section above the "notifications" section.
This way, the issue will be closed once this PR is merged.

**Comment by AntoineVDV** [comment]
> Hi @meng-odoo :) You can link issues and branches in their "development" section above the "notifications" section. This way, the issue will be closed once this PR is merged.

I don't think that it'll work because robodoo closes (red) PRs instead of merging (purple) them. It's still interesting which issues are linked to the PR, though.

File: content/applications/productivity/mail_plugins/gmail.rst [hunk 1]
```diff
@@ -116,8 +116,11 @@ Delete the following three lines of text from the :file:`login.ts` file:
 This removes the `odoo.com` domain constraint from the Gmail Plugin program.
 
 Next, in the ZIP file, go to :menuselection:`mail-client-extensions-master --> gmail`, and open the
-file called :guilabel:`README`. Follow the instructions in the :guilabel:`README` file to push the
-Gmail Plugin files as a Google Project.
+file called :guilabel:`appsscript.json`. In the :guilabel:`urlFetchWhitelist` section, replace all
+the references to `odoo.com` with the Odoo customer's unique server domain.
+
+Then, in the same :guilabel:`gmail` folder, open the file called :guilabel:`README.md`. Follow the
+instructions in the :guilabel:`README` file to push the Gmail Plugin files as a Google Project.
```
Comment (samueljlieber) [inline content/applications/productivity/mail_plugins/gmail.rst:123]: :guilabel:README.md for completeness

**Review by samueljlieber — CHANGES_REQUESTED** [review]
Hi @meng-odoo 👋 Ive done a technical review of the changes on this doc and I'm approving them once I make my commit as the changes are very minor. Then I will tag it for final review.

I just added `.md` to the :guilabel:README for completeness, fixed one line that broke early, and resized two images that were over 768px wide. 

Thank you 🙂

**Comment by samueljlieber** [comment]
Implemented technical changes in https://github.com/odoo/documentation/commit/62aa895781474ff0df4ccdd881f6fb7cd1572077, and sending to final review with @StraubCreative 🙂

**Review by samueljlieber — APPROVED** [review]

File: content/applications/productivity/mail_plugins/gmail.rst [hunk 2]
```diff
@@ -116,12 +116,15 @@ Delete the following three lines of text from the :file:`login.ts` file:
...
+
+Then, in the same :guilabel:`gmail` folder, open the file called :guilabel:`README.md`. Follow the
+instructions in the :guilabel:`README.md` file to push the Gmail Plugin files as a Google Project.
 
 .. note::
    The computer must be able to run Linux commands in order to follow the instructions on the
-   :guilabel:`README` file.
+   :guilabel:`README.md` file.
 
 After that, share the Google Project with the Gmail account that the user wishes to connect to
```
Comment (StraubCreative) [inline content/applications/productivity/mail_plugins/gmail.rst:129]: 100th char
```suggestion
After that, share the Google Project with the Gmail account that the user wishes to connect to Odoo.
```

**Review by StraubCreative — APPROVED** [review]
Looks good @meng-odoo 👍

**Comment by StraubCreative** [comment]
Small update in [646d600](https://github.com/odoo/documentation/pull/3695/commits/646d6002883cc8c2940636699355c42f3f9d74dd). All checks pass 👍 

@robodoo r+

**Pull request closed without merging**